tests/test_input_output.py

```diff
@@ -273,3 +274,12 @@ def test_copy():
         rtol=1e-10,
         atol=1e-10,
     )
+
+
+def test_save_none(tmpdir_factory):
+    tmpdir = tmpdir_factory.mktemp("none_test")
+    eq = Equilibrium()
+    eq._iota = None
```

Comment:
`eq._iota = None` will be redundant with the latest changes on `constrain-toroidal-current-density`, which default to vacuum profiles with `iota  = None`

Reply:
yeah it's just something to make sure this test actually tests the correct thing regardless of what defaults we pick

Reply:
makes sense